src/main/java/calculation/CalculateCoordinatesInterpolationRoutePoints.java

```diff
@@ -80,6 +80,7 @@
       }
     }
     for (int i = 0; i < newCalculatedLat.size(); i++) {
+      System.out.println(newCalculatedLat.get(i) + "," + newCalculatedLong.get(i) + ";");
```

Comment:
you can remove, replcae with log or move to some kind of "debug" dependent logger method